python/lsst/pipe/analysis/analysis.py

```diff
@@ -356,7 +356,7 @@ def plotAgainstMagAndHist(self, log, filename, stats=None, camera=None, ccdList=
         inLimits &= self.data[dataType].quantity > self.qMin
         if self.data[dataType].quantity.any():
             if len(self.data[dataType].quantity[inLimits]) < max(1.0, 0.35*len(self.data[dataType].quantity)):
-                log.info("No data within limits...decreasing/increasing qMin/qMax")
+                log.info("plotAgainstMagAndHist: No data within limits...decreasing/increasing qMin/qMax")
```

Comment:
This is fine. An alternative might be using the logger names.

Reply:
Oh, I'd like to do this the "right" way if there is one!  I'm unfamiliar with this...I'll do some digging, but if you have an example of how to do this handy, that would be great.

Reply:
Hmmm...I tried `log.getName()`, but that, of course, prints the name of the parent task, e.g. `visitAnalysis`.  I don't see another (easy) way of getting at the function name (to which the "logger" was passed).

Reply:
I was thinking to get a logger with the name `visitAnalysis.plotAgainstMagAndHist` and so on.  Maybe `log.getChild("plotAgainstMagAndHist")` could work here.